infra/build/functions/fuzzbench.py

```diff
@@ -28,21 +28,36 @@
 FUZZBENCH_PATH = '/fuzzbench'
 
 
-def get_engine_project_image(fuzzing_engine, project):
+def get_engine_project_image_name(fuzzing_engine, project):
   """Returns the name of an image used to build |project| with
   |fuzzing_engine|."""
   return f'gcr.io/oss-fuzz-base/{fuzzing_engine}/{project.name}'
 
 
-def get_env(project, build):
+def get_ood_image_name(fuzzing_engine, project):
+  """Returns the name of an OSS-Fuzz on Demand image."""
+  return f'us-central1-docker.pkg.dev/oss-fuzz/unsafe/ood/{fuzzing_engine}/{project.name}'
```

Comment:
We should probably abstract this somewhere. Maybe leave a todo if you don't want to do it in this PR.